packages/conf-ppl/conf-ppl.1/opam

```diff
@@ -0,0 +1,17 @@
+opam-version: "1"
+author: "R. Bagnara, P. M. Hill, E. Zaffanella, A. Bagnara et. al."
+maintainer: "francois.berenger@inria.fr"
+homepage: "http://bugseng.com/products/ppl/"
+license: "GPL"
+build: [
+  ["sh" "-c" "cc test.c -lppl_c -lppl -lgmpxx -lgmp"]
```

Comment:
Some of the link flags (e.g. `-lgmpxx` and `-gmp` don't appear to be required to build the test program.